platform/android/export/export_plugin.cpp

```diff
@@ -2431,6 +2431,20 @@ String EditorExportPlatformAndroid::get_apksigner_path(int p_target_sdk, bool p_
 	String java_sdk_path = EDITOR_GET("export/android/java_sdk_path");
 	if (!java_sdk_path.is_empty()) {
 		OS::get_singleton()->set_environment("JAVA_HOME", java_sdk_path);
+
+#ifdef UNIX_ENABLED
+		// Check for the `java` command.
+		String java_path = get_java_path();
+		if (!FileAccess::exists(java_path)) {
+			print_error("Unable to find 'java' command using the Java SDK path specified in Editor Settings.");
+			return "<FAILED>";
+		} else {
```

Comment:
We already validate this in https://github.com/godotengine/godot/blob/e5ccaa79e2a2d5551369bd6cc1f8ed654a791275/platform/android/export/export_plugin.cpp#L2802, is there a need to repeat the validation here as well?